### DISCUSSION TITLE
Retours outil d'affichage

### DISCUSSION THREAD

**Discussion by paulineeyherabide** [betagouv/ma-cantine#1124] [Retours utilisateur·rice]
1/03 : Une utilisatrice m'a fait la remarque suivante - Concernant la phrase "Sur les ... repas servis aux convives, pour l’année 2021, voici la répartition, en valeur d’achat, des produits bio, de qualité et durables (liste de labels ci-dessous) utilisés dans la confection des repas", il y a le détail "par jour" au dessus mais pas dans cette ligne de texte. Peut-être peut-on ajouter "sur les .. repas journaliers servis... voici la répartition annuelle" ?

**Comment by gauthierandre** [comment]
Sur ce sujet, je me permets de relancer une vieille remarque de plusieurs utilisateurs à propos du format **camembert**.
Avantage : très visuel
Défaut : laisse suggérer une marge de progression très importante. Ex, je suis à 20%, la zone grisée peut faire comprendre que tout doit passer au vert.

**Reply by hfroot** [reply]
Qu'est-ce que vous pensez de reprendre la charte en colonnes de la page de publication ?
![Screenshot from 2022-04-13 11-50-37](https://user-images.githubusercontent.com/9282816/163152380-738a0649-72ee-46c8-9e5c-1f859912bab2.png)

**Comment by gauthierandre** [comment]
S'il n'y en a qu'un, ne serait-ce pas étrange ?
Peut-être qu'on peut penser un camembert plus adapté au 50%
au lieu d'avoir %total et les autres, ça peut être %total/2 (car c'est 50% de bio/durable) et %de bio et %de durable et on aurait, en gros par ex : 90% de bio et durable puisqu'on serait sur les achats HT / 2

**Comment by paulineeyherabide** [comment]
Plutôt OK avec André sur le format camembert (AOP bien sûr) ;)
